tests/testthat/test-lua_dynamic_referencing.R

```diff
@@ -0,0 +1,114 @@
+test_that("lua numbering for pdf", {
+  ## this test demonstrates how to use lua filter incl in inst/lua for dynamic labeling
+  ## does not depend on valtools functions for dynamic labeling
+  skip_if_not_installed("valtools")
+  skip_on_os("mac") # not sure if it is os or gh action config
```

Comment:
Do we want to enable testing of this? assuming it runs for you, but would be beneficial to have the tests able to be run.